### PR TITLE
Login Title

### LINKED ISSUE #8027: Login screen
We should find another title for the Login screen than `Use a local account to log in.`

### PR DESCRIPTION
**Pull request by agriffard**
Use the name of the site in the Login title.

image

Fixes #8027

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Users/Views/Account/Login.cshtml [hunk 1]
```diff
@@ -23,7 +23,7 @@
     {
         <div class="col-md-6 @(loginProviders.Count == 0 ? "offset-md-3" : String.Empty)">
             <form asp-controller="Account" asp-action="Login" asp-route-returnurl="@ViewData["ReturnUrl"]" method="post" class="auth-form">
-                <h4>@T["Use a local account to log in."]</h4>
+                <h4>@T["Log in to {0}", Site.SiteName]</h4>
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Users/Views/Account/Login.cshtml:26]: Log into

File: test/Functional/cms-tests/cypress/integration/headless-test.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('Headless Recipe test', function () {
 
     it('Displays the login screen when accessing the root of the Headless theme', function(){
         cy.visit(`${tenant.prefix}`);
-        cy.get('h4').should('contain.text', 'Use a local account to log in.');
+        cy.get('h4').should('contain.text', 'Log in');
```
Comment (hishamco) [inline test/Functional/cms-tests/cypress/integration/headless-test.js:14]: Same here

File: src/OrchardCore.Modules/OrchardCore.Users/Views/Account/Login.cshtml (hunk 1 above)
Comment (agriffard) [reply src/OrchardCore.Modules/OrchardCore.Users/Views/Account/Login.cshtml:26]: https://helpcenter.veeam.com/docs/styleguide/tw/log_in_to.html

> the correct spelling is log in to not log into

**Review by hishamco — APPROVED** [review]
If the spelling is correct so LGTM

**Comment by hishamco** [comment]
Why Linux build is failing?!!

**Comment by Skrypt** [comment]
Can you change the external login title too while we're at it?

**Comment by agriffard** [comment]
> Can you change the external login title too while we're at it?

Change the message? What would it be?

**Comment by Skrypt** [comment]
Use another service to log in to [TenantName].

**Comment by Skrypt** [comment]
I don't like the fact that it wraps on 2 lines because of the long title but it is what it is. LGTM.

**Pull request merged**